sim/simgui-window.json.orig

```diff
@@ -0,0 +1,79 @@
+{
```

Comment:
Do we have a good reason to check in a window file at all? The WPILib template specifically ignores all the window files. https://github.com/wpilibsuite/vscode-wpilib/blob/ccc008aef0ad04ddcd2595edb4d982250504df30/vscode-wpilib/resources/gradle/shared/.gitignore#L172

Reply:
Let's remove it! Problem solved!